File: 백준/Bronze/5086. 배수와 약수/배수와 약수.c
#include <stdio.h>

int main(){
    int a;
    int b;
    
    while(1){
        scanf("%d %d", &a, &b);
        if(a==0&&b==0) break;
        if(b%a==0){
            printf("factor\n");
        }
        else if(a%b==0){
            printf("multiple\n");
        }
        else printf("neither\n");
    }
    return 0;
}

File: 백준/Bronze/5086. 배수와 약수/배수와 약수.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner input = new Scanner(System.in);
        
       int first, second;
        
        while(true){
            first = input.nextInt();
            second = input.nextInt();
        
            if((first==0)&&(second==0)) {
                break;
              }
              
        if(second%first == 0){
            System.out.println("factor");
            
        }
        else if(first%second == 0){
            System.out.println("multiple");
        }
        //second%first == 0 -> factor
        //first%second == 0 -> multiple
        else
            System.out.println("neither");
        }  
        
        input.close();
        
    }
    
}
